Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -3341,13 +3354,13 @@ void MediaPlayerPrivateGStreamer::createGSTPlayBin(const URL& url)
     if (elementId.isEmpty())
         elementId = "media-player"_s;
 
-    auto type = isMediaSource() ? "MSE-"_s : isMediaStream ? "mediastream-"_s : isBlob ? "blob-" : ""_s;
+    ASCIILiteral type = isMediaSource() ? "mse"_s : isMediaStream ? "mediastream"_s : isBlob ? "blob"_s : "media"_s;
```

Comment:
Hm this media prefix isn't great, can lead to names like media-media-player-0 :) Maybe `regular` is better?

Reply:
I agree.